config/settings/production.py

```diff
@@ -132,31 +132,40 @@ class MediaRootS3Boto3Storage(S3Boto3Storage):
 # ------------------------------------------------------------------------------
 # https://docs.djangoproject.com/en/dev/ref/settings/#default-from-email
 DEFAULT_FROM_EMAIL = env(
-    "DJANGO_DEFAULT_FROM_EMAIL", default="Sprint Management Application <noreply@sprints.opencraft.com>"
+    "DJANGO_DEFAULT_FROM_EMAIL", default="Sprints <noreply@sprints.opencraft.com>"
 )
 # https://docs.djangoproject.com/en/dev/ref/settings/#server-email
 SERVER_EMAIL = env("DJANGO_SERVER_EMAIL", default=DEFAULT_FROM_EMAIL)
 # https://docs.djangoproject.com/en/dev/ref/settings/#email-subject-prefix
 EMAIL_SUBJECT_PREFIX = env(
-    "DJANGO_EMAIL_SUBJECT_PREFIX", default="[Sprint Management Application]"
+    "DJANGO_EMAIL_SUBJECT_PREFIX", default="[Sprints]"
 )
+# https://docs.djangoproject.com/en/dev/ref/settings/#email-backend
```

Comment:
Why are we using 2 email backends? 
Are there any benefits to this? 
Will we be using Django's email backend or Anymail as default?

Reply:
Anymail (Mailgun) support was included in the generated part of this project, but, since we want to handle it differently, I made `django.core.mail.backends.smtp.EmailBackend` the default one with an option to easily switch between them. Do you think it makes sense or should we remove Anymail completely from here?

Reply:
@Agrendalath I think that it doesn't hurt to leave it here! We can use it to set up SES in the future, instead of using Gmail accounts or the ops email.